.github/policies/resourceManagement.yml

```diff
@@ -349,18 +349,28 @@ configuration:
       - or:
         - payloadType: Issues
       - isAction:
-          action: Opened
+          action: Labeled

```

Comment:
💡Observation share only: Worth to add context for this change of table please.

Reply:
Added